open-craft/cross-section-link-report
# feat: script to generate report

## Description

Script to generate report about the prevelance of cross-sectional report

## Testing instructions

Check the [README](https://github.com/open-craft/cross-section-link-report/blob/kaustav/generate_report_script/README.md)

## Other information

OpenCraft internal ticket : BB-7352
